package.json

```diff
@@ -15,6 +17,11 @@
     "docker:down": "docker-compose down"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.4.2",
```

Comment:
Rmb to remove unneeded dependencies

Reply:
Removed in new commit

Reply:
we still using ws and tls?

Reply:
I think we should add them when we need them. At the very least not in this pr

Reply:
Ok I can remove them for this pr